### ISSUE TITLE
Association between travel and the variants

### ISSUE THREAD

**Issue by ldnorris** [Lawreros/capstone#7]
Very simple, but I'm thinking about including a tab that just looks at whether or not there's an association between travel and the variants -- like by doing a simple chi-square test or showing a correlation matrix or something

**Comment by Alexa-Malina** [comment]
FYI- this is travel outside the district (not outside Germany), so if they are working in the next big city, this counts as travel

**Comment by ldnorris** [comment]
Got it, didn't realize that. I suppose it could still be interesting to include though

**Comment by ldnorris** [comment]
travel was infrequent, so instead I'm running chi-square tests and showing mosaic plots of age, sex, and condition

**Comment by Alexa-Malina** [comment]
@ldnorris 
"Condition was defined as dead if person was killed from COVID-19 infection." should be changed. 
We don't have the cause of death. 

"Condition was defined as dead if person died within the observation period"

**Comment by ldnorris** [comment]
@Alexa-Malina good catch, just changed it. Once the pull request is approved it should update in the master
